modules/base-laws/src/test/scala/io/janstenpickle/trace4cats/base/context/laws/UnliftTests.scala

```diff
@@ -0,0 +1,11 @@
+package io.janstenpickle.trace4cats.base.context.laws
+
+import cats.~>
+import org.scalacheck.Cogen
+
+class UnliftTests extends BaseSuite {
```

Comment:
And this class (if needed) should be renamed, IMO. It tests only some specific instances of Unlift.